docs/integration.md

```diff
@@ -122,7 +122,7 @@ Also you could use interactive rebase (```git rebase -i HEAD~10```) to delete co
 Check that you're on correct branch:
 ```git status```
 And pull changes from upstream (if any):
-```git pull --rebase upstream master```
+```git pull --rebase upstream controlplane```
```

Comment:
All test job are failing because you are now rebasing on controlplane branch instead of master.